README.md

```diff
@@ -79,8 +79,8 @@ To run the project, execute the following command:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **Add a animation**
-- [ ] **Add a more style**
+- [ ] **Working on it**
```

Comment:
Great job @Hassaanjbaig-code 🤝

- [ ] [Optional] Kindly remove the live demo link from your readme file for now since the project is not yet deployed and the link is broken.